### PR TITLE
Update README_migration.md

### PR DESCRIPTION
**Pull request by nicofrand**

### CODE REVIEW

File: README_migration.md [hunk 1]
```diff
@@ -34,20 +34,26 @@ Most of the previous libs in the libs directory of the cozy/konnectors repositor
...
+| models/folder | models.folder |
 
 How to use it ? We will take the example of baseKonnector and models.bill
 
 ```javascript
 const {baseKonnector, models} = require('cozy-konnector-libs')
+
+//or
+
+const {log, baseKonnector, filterExisting, saveDataAndFile, models} = require('cozy-konnector-libs')
```
Comment (doubleface) [inline README_migration.md]: What is the difference here? You could add log, filterExisting, saveDataAndFile to the previous line. Why "or"?

File: README_migration.md [hunk 2]
```diff
@@ -65,4 +71,4 @@ https://github.com/cozy/cozy-konnector-bouyguestelecom.git
 
 This template repository is delivered with sample package.json, manifest.konnectors and README.md files. You need to update them with information specific to your konnector like : name of the project, dependencies, needed permissions, description of the project.
 
-## You can test your konnector following the procedure described in [./README.md]!
+## You can test your konnector following the procedure described in the [./README.md](README)!
```
Comment (doubleface) [inline README_migration.md]: [README](./README.md) will work better

File: README_migration.md (hunk 1 above)
Comment (nicofrand) [inline README_migration.md]: No difference, the first example was not immediately clear to me.

Comment (nicofrand) [inline README_migration.md]: I did not want to force to include all the methods.

File: README_migration.md (hunk 2 above)
Comment (nicofrand) [inline README_migration.md]: Both work, I just fixed the link

**Pull request merged**